src/Console/Commands/StaticWarm.php

```diff
@@ -188,7 +224,7 @@ protected function customRoutes(): Collection
                 return $route->getActionName() === $action && ! Str::contains($route->uri(), '{');
             })
             ->map(function (Route $route) {
-                return URL::tidy(Str::start($route->uri(), config('app.url').'/'));
+                return Facades\URL::tidy(Str::start($route->uri(), config('app.url').'/'));
```

Comment:
Was there a reason for this change?

Reply:
Oh just to be consistent, as al the other facades are referenced like that in this file.

Reply:
> Ah where we use Facades\Entry:: for example would be because another Entry was already imported.

Yes I got that. No problem changing it back to what is was though, no attachments here 😄